GIS.md

```diff
@@ -0,0 +1,198 @@
+# Why Gnome-Initial-Setup
+There are several use cases where it makes sense for system installation and user creation to not happen at the same 
+time. In these instances, it is useful to have a way to skip the user creation step during installation and for it to 
+be handled later, perhaps even by a different user.
+
+To accommodate this workflow Ubuntu ships with Gnome Initial Setup, a program that can safely handle user account 
+creation and configuration. Gnome Initial Setup is started by default when a system is booted and GDM is unable to
+detect any user accounts. Instead of taking you to the login screen, a special Gnome Initial Setup user session is started.
+
+This guide aims to highlight some of the ways a system can be provisioned such that Ubuntu is installed, but no user 
+account is created, allowing user account creation to be passed off to Gnome-Initial-Setup.
+
+# Triggering Gnome-Initial-Setup via the new Flutter installer and a whitelabel.yaml
+
+The new [Ubuntu Desktop Bootstrap](https://github.com/canonical/ubuntu-desktop-provision) Flutter installer can be 
+given a `whitelabel.yaml` file to customize its appearance and behavior. One thing you can configure is the `mode` you 
+would like to run the installer in. 
+
+The `oem` skips user creation during installation, meaning that user creation will be handled by Gnome Initial Setup 
```

Comment:
```suggestion
The `oem` mode skips user creation during installation, meaning that user creation will be handled by Gnome Initial Setup 
```

Do we need to mention that `oem` mode might have other side-effects as well? Currently it also skips the time zone page, which might fall under the previously mentioned "account configuration".

Reply:
Ah, good point, let me add that :slightly_smiling_face: